core/templates/pages/community-dashboard-page/contributions-and-review/contributions-and-review.directive.ts

```diff
@@ -386,13 +378,41 @@ angular.module('oppia').directive('contributionsAndReview', [
               }
             ];
             UserService.getUserInfoAsync().then(function(userInfo) {
-              ctrl.isAdmin = userInfo.isAdmin();
               ctrl.userIsLoggedIn = userInfo.isLoggedIn();
-              ctrl.userDetailsLoading = false;
-              if (ctrl.isAdmin) {
-                ctrl.switchToReviewTab(ctrl.SUGGESTION_TYPE_QUESTION);
-              } else if (ctrl.userIsLoggedIn) {
-                ctrl.switchToContributionsTab(ctrl.SUGGESTION_TYPE_QUESTION);
+              ctrl.userDeatilsLoading = false;
+              if (ctrl.userIsLoggedIn) {
+                UserService.getUserCommunityRightsData().then(
+                  function(UserCommunityRights) {
+                    var userCanReviewTranslationSuggestionsInLanguages = (
+                      UserCommunityRights
+                        .can_review_translation_for_language_codes);
+                    var userCanReviewQuestionSuggestions = (
+                      UserCommunityRights.can_review_questions);
+                    if (userCanReviewQuestionSuggestions) {
+                      ctrl.reviewTabs.push({
+                        suggestionType: ctrl.SUGGESTION_TYPE_QUESTION,
+                        text: 'Review Questions'
+                      });
+                    }
+                    if (
+                      userCanReviewTranslationSuggestionsInLanguages
+                        .length > 0) {
+                      ctrl.reviewTabs.push({
+                        suggestionType: ctrl.SUGGESTION_TYPE_TRANSLATE,
+                        text: 'Review Translations'
+                      });
+                    }
+                    if (ctrl.reviewTabs.length > 0) {
+                      ctrl.canReviewTranslation = true;
+                      ctrl.reviewTabActive = false;
```

Comment:
reviewTabIsActive

Also I'm a bit confused. Why is this inactive when reviewTabs.length > 0 and active otherwise? It seems counterintuitive.